### ISSUE TITLE
add license

### ISSUE THREAD

**Issue by SkyBulk** [imnotannamaria/Tipfy#1]
it's a very interesting project!

Would you consider adding a license to this repository to specify under which terms this code can be used?

**Comment by imnotannamaria** [comment]
Yes, but not in this first version, i am working on the definitive version, and in this version I will a license.
